test/driver/pg_schema_test.js

```diff
@@ -1,4 +1,5 @@
 var vows = require('vows');
+var async = require('async');
```

Comment:
Please don't use async, I'm currently working on kick this dependency out of the project. The whole project will rely on promises in the future.

I have ported your changes to use bluebird in the master repo, but please consider this for future contributions.